spyder/app/tour.py

```diff
@@ -815,6 +832,11 @@ def mousePressEvent(self, event):
 #            if clicked_widget == self.label_current:
 #            self.context_menu_requested(event)
 
+    def focusOutEvent(self, event):
+        """Reimplement Qt method."""

```

Comment:
This is probably a super minor comment.

But the convention we use is 

`Override Qt method.`